### PR TITLE
Jessica Robinson - HW01

### PR DESCRIPTION
**Pull request by jessrobinson42**
Overall, I feel a got better sense of both Git and Markdown from this process. Prior to the homework, I was a little confused about setting up Git on my personal computer. Everything has seemed to work so far, but I'm not sure I understand why as fully as I would like. The flow between RStudio and Git, once I got the hang of it, was relatively straightforward. Markdown was not too hard to work with, which I think is because I have used R Markdown a bit in other classes before. However, I still found myself having some minor issues. For instance, despite the section headings showing up correctly on my preview in R, they failed to show up correctly on Git, even after multiple troubleshooting steps, leading me to use a different format for section headings. GitHub Help and Stack Overflow were helpful references in resolving these kinds of issues.

### CODE REVIEW

**Review by ehuppert — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |      x     |              |            |
| **Coding strategy**         |      x     |              |            |
| **Presentation**            |  x         |              |            |
| **Achievement, creativity** |           |       x       |            |
| **Ease of access**          |    x       |              |            |

Remarks:

* Elaborate on above, especially for "needs work."
Nothing "needs work" because you did a good job and the code is easy to read. Your page is simple and clear! I put satisfactory instead of excellent for the achievement, creativity line item because I didn't see anything unique or above and beyond experimentation, but I think thats really hard to do with this type of About Me assignment. Good job experimenting with lists, headers, bold, the image, and links!

* Some specific praise?
I thought the idea to include social media links as more “about me” information was a great idea! Your format was nice and easy to read.

* Something I learned?
I enjoyed reading your thoughtful reflection. I appreciate that you mentioned the sources you went to for help and this was helpful for me! 

* Specific constructive criticism?
I really liked that you included an image, but I think it could have gone after your intro with your name or had a caption. It was a little out-of-the-blue as the first thing to pop up on the file, but this is just an aesthetic comment. Everything worked well.

* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
It sounds like your issue with the headings from your reflection wasn’t quite resolved (you changed the format rather than fixing the problem if I am understanding correctly). I don’t know the answer to this, but I had similar issues with things showing up differently in GitHub and R and it was because of my output. I had put the output as html instead of a GitHub document. It sounds like you have used R before so maybe you had a different output too? This could be something to look at to fix the problem.

**Review by domdicarlo — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |     x     |              |            |
| **Coding strategy**         |     x     |              |            |
| **Presentation**            |     x     |              |            |
| **Achievement, creativity** |     x     |              |            |
| **Ease of access**          |     x     |              |            |

Remarks:

* Elaborate on above, especially for "needs work."
  * Overall, I think you did a great job with this assignment. Your bio and intro to the course is short and sweet, and your picture reminds me that I know you from the Macbeth play. You make use of a variety of markdown styling techniques such as headers, bolding, and links. 
* Some specific praise?
  * Your use of a picture got you an _Excellent_ in terms of achievement, since I did not bother to figure out how to do this. Also the alternative way of making headings was cool to see.
* Something I learned?
  * After checking the code I learned how to add in a picture via markdown. I also learned another way to do section headings in md, since as you mentioned your previous ones weren't showing up correctly
* Specific constructive criticism?
  * Can't really think of anything for this assignment
* Something I know and that you, my peer, might like to know because it is relevant to something you struggled with.
  * Just ditto on the minor issues thing, coding will always have little annoyances like that where bugs might show up in one place and not another, but it gets better with time as the syntaxes become second nature.

File: README.md [hunk 1]
```diff
@@ -1,2 +1,23 @@
-# hw01
```
Comment (jfan3) [inline README.md:1]: Nice!

**Review by jfan3 — COMMENTED** [review]
Evaluation
----------------------------------------------------
| Topic                       | Excellent | Satisfactory | Needs Work |
|-----------------------------|-----------|--------------|------------|
| **Coding style**            |x           |              |            |
| **Coding strategy**         |x           |              |            |
| **Presentation**            |x           |              |            |
| **Achievement, creativity** |           |x              |            |
| **Ease of access**          |x           |              |            |

Remarks:

* Nice work!

**Pull request closed without merging**